.vscode/settings.json

```diff
@@ -8,8 +8,10 @@
     // https://github.com/Microsoft/vscode-react-native/blob/master/doc/intellisense.md .
     "flow.useNPMPackagedFlow": true,
 
-    // Format on save (with prettier-eslint).
     "[javascript]": {
+        // Format JS code with prettier-eslint.
+        "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
I wonder if we can automatically pull in <5.0.0 of this extension. ISTR things break for us on v5.0.0. I'm using v4.7.0 and it's been working; see [discussion](https://chat.zulip.org/#narrow/stream/48-mobile/topic/Add.20Emoji.20Reaction.20Button/near/1152687).

I guess one important question: what version have you been using? 😛

Reply:
I've been using v4.7.0 too.

The place we specify to use this extension is actually a different file, `.vscode/extensions.json`.

Hmm, one thing we should perhaps do is: if we don't find a way to put a version on that recommendation, then delete the `editor.formatOnSave` line that comes after this one. That way if you have a wrong version (that doesn't use prettier-eslint, and therefore gets some formatting wrong), it won't be as obnoxious.

Reply:
I investigated a bit, and there doesn't seem to be a way to specify versions in `extensions.json`.

Specifically, there is a syntax for doing so on the command line, which is otherwise similar:
```
$ code --install-extension esbenp.prettier-vscode@4.7.0
```

But if I try writing `"esbenp.prettier-vscode@4.7.0"` in the `recommendations` list of `extensions.json`, instead of simply `"esbenp.prettier-vscode"`, then VS Code gives me a warning popup reading:
> The 1 extension(s) below, in workspace recommendations have issues: esbenp.prettier-vscode@4.7.0 (bad format)
> Expected: <provider>.<name>

So I think I'll do this:

> then delete the `editor.formatOnSave` line that comes after this one. That way if you have a wrong version (that doesn't use prettier-eslint, and therefore gets some formatting wrong), it won't be as obnoxious.